lingetic-nextjs-frontend/app/error.tsx

```
@@ -0,0 +1,40 @@
+      <div className="max-w-md w-full space-y-8 p-10 bg-white shadow rounded-xl">
+        <div className="text-center">
+          <h1 className="text-3xl font-bold text-[#374151] mb-2">
+            Oops! Something went wrong.
+          </h1>
+          <p className="text-[#dc2626] mb-8">
+            We encountered an unexpected error.
+          </p>
+        </div>
+        <div className="flex flex-col space-y-4">
+          <button
+            autoFocus
```

Biome (1.9.4)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)